Automattic/wp-calypso
# Stats: Update how SVG is included on Stats purchase page

Related to #80269

## Proposed Changes

* removing SVG from JS to a separate file to reduce JS file size

## Testing Instructions



* navigate to a live branch and go to `/stats/purchase/:siteSlug`
* verify that SVG on the right side looks the same on the live branch and in production
* verify that the `new features` icon updates properly when selecting the free plan
* verify that commercial plan has celebration SVG on top of the regular one

## Pre-merge Checklist
